include/meltpooldg/simulations/melt_front_propagation/melt_front_propagation.hpp

```diff
@@ -104,7 +104,7 @@ namespace MeltPoolDG::Simulation::MeltFrontPropagation
           refinements[0] = 3;
           // create mesh
           const Point<2> left(0, 0);
-          const Point<2> right(x_max, y_max);
+          const Point<2> right(x_max, -z_max);
```

Comment:
```suggestion
          const Point<2> left(0, -z_max);
          const Point<2> right(x_max, 0);
```
I guess deal.II will re-orient the cells internally, but the notion is normally `left-bottom-front` and `right-top-back`.

Reply:
@peterrum: Do you see an issue here?

Reply:
I already applied @peterrum 's suggestion